bip-0158/gentestvectors.go

```diff
@@ -37,7 +37,7 @@ var (
 		{49291, "Tx pays to empty output script"},
 		{180480, "Tx spends from empty output script"},
 		{926485, "Duplicate pushdata 913bcc2be49cb534c20474c4dee1e9c4c317e7eb"},
-		{987876, "Coinbase tx has unparseable output script"},
+		{987876, "Coinbase tx has unparsable output script"},
```

Comment:
"unparseable" is not a typo but a valid alternative spelling.

Reply:
Indeed, so somewhat false positive, let me know if I should revert.